Dockerfile

```diff
@@ -5,7 +5,8 @@ WORKDIR /opt/test-runner
 COPY project/ project/
 COPY src/ src/
 COPY build.sbt build.sbt
-RUN sbt compile
+RUN sbt assembly
+RUN chmod +x target/scala-2.12/TestRunner-assembly-0.1.0-SNAPSHOT.jar
```

Comment:
Set executable permissions to allow running the jar

Reply:
the intention of the PR was not this. The meaning of using an assembled fat jar is to deploy it together with the test-runner and use that as an executable to run the uploaded exercises, not to generate it upon running the test-runner, because we'd be back to point one.

Reply:
But we're not doing that. Were building the executable in the _Dockerfile_, which is only done when the test runner is deployed. For each test run, we're using the compiled test runner (see https://github.com/exercism/scala-test-runner/pull/14/files#diff-00a80c5821edea2ebf676056aa4c9a24e57379ef52cefecb2ccffaaf4cc362c9R57). The code in the Dockerfile is _not_ executed.